scu-de/src/fiskaltrust.Middleware.SCU.DE.SwissbitCloudV2/Services/SwissbitCloudV2ApiProvider.cs

```diff
@@ -0,0 +1,183 @@
+
+            var response = await _httpClient.PostAsync($"/api/v1/tse/{transactionType}", new StringContent(jsonPayload, Encoding.UTF8, "application/json"));
+            var responseContent = await response.Content.ReadAsStringAsync();
+            
+            return JsonConvert.DeserializeObject<TransactionResponseDto>(responseContent);
+        }
+
+        public async Task<List<string>> GetClientsAsync()
+        {
+            var response = await _httpClient.GetAsync($"/api/v1/tse/clients");
+            var responseContent = await response.Content.ReadAsStringAsync();
+            
+            return JsonConvert.DeserializeObject<List<string>>(responseContent);
+        }
+        public async Task<TseDto> GetTseStatusAsync()
+        {
+            var response = await _httpClient.GetAsync($"/api/v1/tse");
+            var responseContent = await response.Content.ReadAsStringAsync();
+            
+            return JsonConvert.DeserializeObject<TseDto>(responseContent);
+        }
+
+        public async Task<TseDto> DisableTseAsync()
+        {
+            await _httpClient.PostAsync("/api/v1/tse/disableSecureElement", new StringContent("", Encoding.UTF8, "application/json"));
+            
+            return await GetTseStatusAsync();
+        }
+
+        public async Task<List<int>> GetStartedTransactionsAsync()
+        {
+            var response = await _httpClient.GetAsync($"/api/v1/tse/transactions");
+            var responseContent = await response.Content.ReadAsStringAsync();
+           
+            return JsonConvert.DeserializeObject<List<int>>(responseContent);           
+        }
+
+        public async Task CreateClientAsync(ClientDto client)
+        {
+            var clientDto = new ClientDto { ClientId = client.ClientId };
+            var jsonPayload = JsonConvert.SerializeObject(clientDto, _serializerSettings);
+
+            await _httpClient.PostAsync($"/api/v1/tse/registerClient", new StringContent(jsonPayload, Encoding.UTF8, "application/json"));
+            
+        }
+
+        public async Task DeregisterClientAsync(ClientDto client)
+        {
+            var jsonPayload = JsonConvert.SerializeObject(client, _serializerSettings);
+
+            await _httpClient.PostAsync($"/api/v1/tse/deregisterClient", new StringContent(jsonPayload, Encoding.UTF8, "application/json"));            
+        }
+
+        public async Task<ExportDto> StartExportAsync()
+        {
+            var openExports = await GetExportsAsync();
+            if (openExports.Count > 0)
+            {
+                _logger.LogInformation($"There is already an open export with id {openExports[0].Id}. Returning this export, as only one export can be processed by Swissbitcloudv2 at a time." +
+                    $"To start a new export the old export data must be deleted and stored with a daily closing. If another daily closing is happening on another queue for this tse, data will be saved and deleted on that queue.");
```

Comment:
It's probably not very clear to a user/implementor how this should be done 🤔
I'm not sure what I need to do if I see this message...